python-types.c

```diff
@@ -1460,7 +1469,7 @@ tgl_Msg_repr(tgl_Msg *self)
 #else
   ret = PyUnicode_FromFormat("<tgl.Msg id=%ld, flags=%d, mention=%R, out=%R, unread=%R, service=%R, src=%R, "
                              "dest=%R, text=%R, media=%R, date=%R, fwd_src=%R, fwd_date=%R, reply_id=%R, reply=%R>",
-                             self->msg->id, self->msg->flags,
+                             self->msg->permanent_id.id, self->msg->flags,
```

Comment:
Fix permanent id here but forget to fix it 4 lines above: https://github.com/Cojad/tg/blob/dc9ee85bd78dd719256d8b43135bf0e537a5adc8/python-types.c#L1468